eng/common/pipelines/templates/steps/verify-links.yml

```diff
@@ -8,7 +8,7 @@ parameters:
   Urls: '(Get-ChildItem -Path ./ -Recurse -Include *.md)'
   BranchReplaceRegex: "^(${env:SYSTEM_PULLREQUEST_SOURCEREPOSITORYURI}.*/(?:blob|tree)/)master(/.*)$"
   BranchReplacementName: "${env:SYSTEM_PULLREQUEST_SOURCECOMMITID}"
-
+  UserAgent: ${env:USERAGENT}
```

Comment:
Should we just default this to empty? Or do you expect that we would set this as a pipeline variable?

Reply:
I plan to set it through pipeline variable as it does not affect all pipelines. Just want to limit use the custom userAgent setting.

Reply:
Some links prefer specific userAgent. We currently do not have the full picture of all scenarios, so I prefer to use pipeline UI variable which is easy to test and modify.

Reply:
If we can get the default of Chrome to work in our usages (I'd test it in the azure-sdk link checking pipeline), then lets  just remove this in the yml. If we decide to keep this we should make the variable name more specific something like VERIFY_LINK_USERAGENT, so that we don't accidently clash with others.

Reply:
Chrome PSUserAgent does not work for the twitter link, I have to switch to "Chrome/87.0.4280.88" to invoke twitter link

Reply:
Working one: 
```Invoke-WebRequest -Method GET -URI "https://www.twitter.com/AzureSDK" -UserAgent "Chrome/87.0.4280.88"```

Default one not working:
```Invoke-WebRequest -Method GET -URI "https://twitter.com/AzureSDK" -UserAgent  "[Microsoft.PowerShell.Commands.PSUserAgent]::Chrome"```

Reply:
Maybe just default to the newer Chrome version in the script as well.